connector/src/main/scala/com/datastax/spark/connector/datasource/CassandraCatalog.scala

```diff
@@ -426,6 +425,28 @@ object CassandraCatalog {
     tableMeta
   }
 
+  //Table or Materialized View Support
+  def getRelationMetaData(connector: CassandraConnector, ident: Identifier): RelationMetadata = {
```

Comment:
Hi! Thanks for the patch! Could you add a test verifying the new behavior? It could be added to `CassandraDataSourceSpec`. Create a view in `beforeClass` and load data in a new test or tests.

Reply:
Added tests